docs/segments/io.rst

```diff
@@ -82,6 +82,18 @@ Extra attributes can be written to the tables via the ``attrs={}`` keyword, all
 
     >>> f.write('new-table.xml', append=True, overwrite=True, attrs={'process_id': 0})
 
+.. note::
+
+   The |python-ligo-lw| library reads and writes files using an updated
+   version of the format compared to :mod:`glue.ligolw` used to. GWpy
```

Comment:
@duncanmmacleod, this is definitely pedantic, but I think maybe this passage can be re-worded?

```suggestion
   version of the ``LIGO_LW`` format compared to :mod:`glue.ligolw`. GWpy
```